docs/source/gdpr/i18n/data_user_agreement.es.rst

```diff
@@ -2,20 +2,20 @@
 
 Spanish
 -------
-(translation courtesy of Dr Maria de la Iglesia)
+(translation courtesy of Dr Maria de la Iglesia and Guiomar Niso)
 
-Acuerdo de usuario de datos para acceder a datos humanos identificables
+Acuerdo de Uso de Datos para el acceso a datos humanos identificables
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Acuerdo de Uso de Datos para el acceso a datos humanos identificables
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

to ensure consistent underlining there (we seemed were 1 character too many before too)